test/CUDADeviceCode/CUDAInclude.C

```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
```

Comment:
You're welcome to add ` // author:  Simeon Ehrig <s.ehrig@hzdr.de> ` also for tests.